library-and-framework-list.json

```diff
@@ -130,6 +130,24 @@
       }
     ]
   },
+  {
+    "artifact": "de.focus-shift:jollyday",
+    "description": "The library to retrieve holiday information for a given year, country and state/region.",
+    "details": [
+      {
+        "minimum_version": "0.21.0",
+        "metadata_locations": [
+          "https://repo1.maven.org/maven2/de/focus-shift/jollyday/",
+          "https://repo1.maven.org/maven2/de/focus-shift/jollyday-core/",
+          "https://repo1.maven.org/maven2/de/focus-shift/jollyday-jaxb/"
+        ],
+        "tests_locations": [
+          "https://github.com/focus-shift/jollyday/blob/main/.github/workflows/build-native.yaml"
```

Comment:
Do you also run any native tests (which is a requirement for the `fully-tested` test level)? (e.g., via `nativeTest` or manually on the executable)? I didn't see any in the workflows directory. If you don't run native tests, that is ok, but we need to adjust the `test_level` to `community-tested`.

Reply:
We run them on every push. The workflow is here https://github.com/focus-shift/jollyday/blob/main/.github/workflows/build-native.yaml and builds are here https://github.com/focus-shift/jollyday/actions/workflows/build-native.yaml

Reply:
I see that you run [some tests on HotSpot](https://github.com/focus-shift/jollyday/actions/runs/6687200507/job/18167554024#step:4:628), but it seems [native binaries are not even built](https://github.com/focus-shift/jollyday/actions/runs/6687200507/job/18167554024#step:4:768):
>Warning:  The requested profile "native" could not be activated because it does not exist.

Are you sure that you have set up the [Native Build Tools](https://graalvm.github.io/native-build-tools/) correctly?

Reply:
I will take a look at it. Thanks for your reply.